src/stores/user.ts

```diff
@@ -1,15 +1,26 @@
 import { defineStore } from "pinia";
 import { useStorage } from "@vueuse/core";
+import { useCurrentUser, Roles } from "@/features/user";
 
 export const useUserStore = defineStore("user", {
   state: () => {
     return {
       preferences: useStorage("preferences", { kanaal: "" }),
+      userData: useCurrentUser(),
```

Comment:
ipv de store zelf usecurrentuser aan te laten roepen, lijkt het me beter om vanuit de user feature de waarde in de store te zetten via een action. 
Als dat niet kan dan misschien usecurrentuser verplaatsen naar een user service in de root.  en dan de user feature voor het onderscheid hernoemen naar Login of autorisatie

Reply:
Laten we hier morgen even over bellen, want dit klinkt voor mij niet logisch.

Reply:
wat langer over nagedacht, zit toch wat in. denk dat het wel gaat lukken.

Reply:
de feature heet nu login. de User en Role types horen bij de store.